apps/api/src/main.ts

```diff
@@ -22,7 +31,8 @@ app.use(session({
   cookie: {
     http: false,
     sameSite: 'lax'
-  }
+  },
+  store
```

Comment:
Are you planning on building onto the session store? If not, why not instantiate here in this file?

Reply:
discussed to keep as is